modules/amd-installing-gpu-operator.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * hardware_accelerators/amd-gpu-operator.adoc
+
+:_content-type: PROCEDURE
+[id="amd-installing-gpu-operator_{context}"]
+= Installing the AMD GPU Operator
+
+As a cluster administrator, you can install the AMD GPU Operator by using the OpenShift CLI and the web console. Use the following procedure to install the AMD community release of the Operator.
+
+Use the following steps to install and configure the AMD GPU Operator.
+
+. Install the xref:../hardware_enablement/psap-node-feature-discovery-operator.adoc#installing-the-node-feature-discovery-operator_node-feature-discovery-operator[Node Feature Discovery Operator].
```

Comment:
@egallen @yevgeny-shnaidman Do we still need lines 13-15 (Install procedures for NFD and KMM)?